api/api-rule-violations-known.list

```diff
@@ -147,6 +147,7 @@ API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Disks
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,GPUs
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Inputs
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,Devices,Interfaces
+API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,DownwardAPIVolumeSource,Fields
```

Comment:
we're trying to avoid new api rule violations moving forward. 

This violation can be avoided by adding a ```//+listType=[atomic|set|map]``` to the field in schema.go.

more details here, https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go#L22

Reply:
Done.